crates/pop-cli/src/commands/up/contract.rs

```diff
@@ -68,6 +68,21 @@ impl UpContractCommand {
 			log::success(result.to_string())?;
 		}
 
+		if !is_chain_alive(self.url.clone()).await? {
+			if !confirm(format!(
```

Comment:
I'd actually allow the user to pass a flag to skip this check. Thinking about CI.

Reply:
I meant... users using pop are the ones that would execute this command in the CI. That's what I meant.

Reply:
You are right, I have added a `skip_confirm` flag.